wagtail/admin/templates/wagtailadmin/pages/edit.html

```diff
@@ -54,7 +54,9 @@ <h1 class="left col9 header-title" title="{% blocktrans with title=page.get_admi
         {% endif %}
 
         <footer>
-            <nav aria-label="{% trans 'Actions' %}">
+            <ul>
+                <li class="footer__container">
+                <nav aria-label="{% trans 'Actions' %}">
```

Comment:
Could you ident this? (Github's "hide whitespace changes" feature works pretty well for reviewing changes like this)